src/plugins/intel_gpu/include/intel_gpu/primitives/shape_of.hpp

```diff
@@ -14,18 +14,6 @@ struct shape_of : public primitive_base<shape_of> {
 
     shape_of() : primitive_base("", {}) {}
 
-    /// @brief Constructs shape_of primitive.
-    /// @param id This primitive id.
-    /// @param input Input primitive id.
-    /// @param output_data_type type of output values. can be i32 and i64.
-    shape_of(const primitive_id& id,
-             const input_info& input,
-             size_t input_rank,
-             const data_types output_data_type,
-             const padding& output_padding = padding())
-        : primitive_base(id, {input}, {output_padding}, {optional_data_type{output_data_type}})
-        , input_rank(input_rank) {}
-
     /// @brief Constructs shape_of primitive.
```

Comment:
As far as I know, the original input rank was added for some reason  in this PR https://github.com/openvinotoolkit/openvino/pull/17912, for static shape. @steve-y  Could you recall the prior issue and whether this removal is okay?

Reply:
@yeonbok This PR has been tested on CVS-110900 for model and functional tests and they worked. And the removal was okay.

Reply:
Thanks for confirmation.